TIL/daily/2025-03-20.md

```
@@ -0,0 +1,483 @@
+final Drawable d = elements[i];  //실제 타입은 런타임에 결정됨
+d.draw(); //실제 구현체의 draw() 메서드가 호출됨.
+```
+
+- **switch 문으로 타입 체크 가능**
+    - 패턴 매칭을 통해서 각 타입을 파악할 수 있다.
+
+```dart
+final Drawable drawable = elements[i];
+
+switch (drawable) {
+	case Rectangle();
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)